CMakeLists.txt

```diff
@@ -5,17 +5,7 @@ set(PROJECT_DESCRIPTION "Open source version of the STMicroelectronics Stlink To
 set(STLINK_UDEV_RULES_DIR "/etc/udev/rules.d" CACHE PATH "Udev rules directory")
 set(STLINK_MODPROBED_DIR "/etc/modprobe.d" CACHE PATH "modprobe.d directory")
 set(LIB_INSTALL_DIR "lib" CACHE PATH "Main library directory")
-
-if(IS_DIRECTORY CMAKE_LIBRARY_PATH)
-else()
-    set(CMAKE_LIBRARY_PATH "lib")
-endif()
-
-if (WIN32)
-    set(STLINK_LIBRARY_PATH ${CMAKE_LIBRARY_PATH} CACHE PATH "Target lib directory")
-else()
-    set(STLINK_LIBRARY_PATH "${LIB_INSTALL_DIR}/${CMAKE_LIBRARY_PATH}" CACHE PATH "Target lib directory")
-endif(WIN32)
+set(STLINK_LIBRARY_PATH "${CMAKE_INSTALL_PREFIX}/${LIB_INSTALL_DIR}" CACHE PATH "Target lib directory")
```

Comment:
Seems that LIB_INSTALL_DIR already /usr/lib64 and CMAKE_INSTALL_PREFIX = /usr.
So lib installed to /usr/usr/lib64/
Need to remove CMAKE_INSTALL_PREFIX from this line.